website/docs/guides/adapter-creation.md

```diff
@@ -1239,8 +1241,9 @@ Breaking this down:
 - Contributor recognition (if applicable)
     <Lightbox src="/img/adapter-guide/6-thank-contribs.png" title="thank yous"/>
 
+## Build a trusted adapter
 
-## Verify a new adapter
+The Trusted Adapter Program exists to allow adapter maintainers to demonstrate to the dbt community that your adapter is trusted to be used in production.
 
 The very first data platform dbt supported was Redshift followed quickly by Postgres (([dbt-core#174](https://github.com/dbt-labs/dbt-core/pull/174)). In 2017, back when dbt Labs (née Fishtown Analytics) was still a data consultancy, we added support for Snowflake and BigQuery. We also turned dbt's database support into an adapter framework ([dbt-core#259](https://github.com/dbt-labs/dbt-core/pull/259/)), and a plugin system a few years later. For years, dbt Labs specialized in those four data platforms and became experts in them. However, the surface area of all possible databases, their respective nuances, and keeping them up-to-date and bug-free is a Herculean and/or Sisyphean task that couldn't be done by a single person or even a single team! Enter the dbt community which enables dbt Core to work on more than 30 different databases (32 as of Sep '22)!
```

Comment:
@dataders : if you could review the "trusted adapter" parts for accuracy, that would be great! 

changes per my chat with amy:
- moved the intro/historical content in "Verify a new adapter" to "Build a trusted adapter"
- deleted "Verify a new adapter"
- deleted other "verify | verified" subsections
- removed/replaced all references to "verified" in guide